cmake/flash/CMakeLists.txt

```diff
@@ -89,9 +89,7 @@ foreach(target flash debug debugserver attach)
   # build is already up to date before west is run.
   set(cmd
     ${CMAKE_COMMAND} -E env
-    PYTHONPATH=${ZEPHYR_BASE}/scripts/meta
-    ${PYTHON_EXECUTABLE}
-    ${ZEPHYR_BASE}/scripts/meta/west/main.py
+    west
```

Comment:
should we maybe use find_program() here?

Reply:
I thought about it, but did not want it to be mandatory. I will add a check elsewhere.

Reply:
find_program should be used.

Using it does not imply it is mandatory.

Using find_program would be consistent with other host tool dependencies and would give the user a consistent way to override it's location (-DWEST=/path/to/west).